pysam/libcalignedsegment.pyx

```diff
@@ -486,38 +486,38 @@ cdef inline object getQualitiesInRange(bam1_t *src,
 
 
 #####################################################################
-## private factory methods
+## factory methods for instantiating extension classes
 cdef class AlignedSegment
-cdef makeAlignedSegment(bam1_t * src, AlignmentFile alignment_file):
+cdef makeAlignedSegment(bam1_t * src, AlignmentHeader header):
```

Comment:
I prefer the convention where there is no space after `*`.  It seems the most common convention in well written C code and what I've been emulating in the BCF code.